app/views/components/_eventbox.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="event-resultbox">
+    <div class="event-resultbox__header">
+        <h2><%= title %></h2>
+    </div>
+    <div class="event-resultbox__datetime">
+        <%= datetime %>
+    </div>
+    <div class="<%=  traintoteach ? 'event-resultbox__greenstripe' : 'event-resultbox__bluestripe' %>"></div>
+    <div class="event-resultbox__content">
+        <p>
+            <%= description %>
+        </p>
+        <div style="<%=  traintoteach ? '' : 'display:none'%>"><h3><div class="event-resultbox__content__icon icon-train-to-teach " ></div> Train to teach event</h3></div>
```

Comment:
These types of conditional styles could be more concise, e.g. `<%= "display: none" unless traintoteach %>`